coderd/database/queries/notificationsinbox.sql

```diff
@@ -21,8 +21,8 @@ SELECT * FROM inbox_notifications WHERE
 -- param limit_opt: The limit of notifications to fetch. If the limit is not specified, it defaults to 25
 SELECT * FROM inbox_notifications WHERE
 	user_id = @user_id AND
-	template_id = ANY(@templates::UUID[]) AND
```

Comment:
The previous query was not working as we want for empty values -
I've done a lot of tests and both template and targets logic were culprits here.